include/RevTracer.h

```diff
@@ -114,24 +119,61 @@ namespace SST::RevCPU{
     } f;
   };
 
-  enum TraceKeyword_t { RegRead, RegWrite, MemLoad, MemStore, PcWrite };
+  enum TraceKeyword_t { 
+    RegRead,       // register read (non-fp)
+    RegWrite,      // register write (non-fp)
+    MemLoad,       // issue load request (simple mem)
+    MemStore,      // issue store request (simple mem)
+    MemhSendLoad,  // issue load request (memh)
+    PcWrite,       // write program counter
+    };
 
   // Generic record so we can preserve code ordering of events in trace
-  //     // register       memory
-  // a;  // reg             adr
-  // b;  // value           len
-  // c;  // origin(TODO)    data (limit 8 bytes)
+  //     // register       memory                   memh
+  // a;  // reg             adr                     adr
+  // b;  // value           len                     len
+  // c;  // origin(TODO)    data (limit 8 bytes)    reg
   struct TraceRec_t {
     TraceKeyword_t key;
-                 // register        memory
-    uint64_t a;  // reg             adr
-    uint64_t b;  // value           len
-    uint64_t c;  // origin(TODO)    data (limited to show 8 bytes)
+                 // register        memory                  memh
+    uint64_t a;  // reg             adr                     adr
+    uint64_t b;  // value           len                     len
+    uint64_t c;  // origin(TODO)    data (limited 8 bytes)  reg
     TraceRec_t() {};
```

Comment:
Are there instances where you want `TraceRec_t` to be uninitialized, and initialize it later (e.g. with aggregate initialization `tracerec = {key, a, b, c}`?) To avoid uninitialized members, I usually use default member initialization, e.g.
```c++
TraceKeyword_t key = -1;
uint64_t a = 0;
uint64_t b = 0;
uint64_t c = 0;
```
Or just remove `TraceRec_t() {}` default constructor, since the other constructor initializes all members.